po/ko.po

```diff
@@ -1,768 +1,756 @@
+#: src/SUMMARY.md:40
 msgid "Type Inference"
-msgstr ""
+msgstr "타입 추론"
 
-#: src/SUMMARY.md:43
+#: src/SUMMARY.md:41
 msgid "static & const"
 msgstr ""
 
-#: src/SUMMARY.md:44
+#: src/SUMMARY.md:42
 msgid "Scopes and Shadowing"
-msgstr ""
+msgstr "스코프(범위)와 쉐도잉"
 
-#: src/SUMMARY.md:45
+#: src/SUMMARY.md:43
 msgid "Memory Management"
-msgstr ""
+msgstr "메모리 관리"
 
-#: src/SUMMARY.md:46
+#: src/SUMMARY.md:44
 msgid "Stack vs Heap"
 msgstr ""
 
-#: src/SUMMARY.md:47
+#: src/SUMMARY.md:45
 msgid "Stack Memory"
-msgstr ""
+msgstr "스택 메모리"
 
-#: src/SUMMARY.md:48
+#: src/SUMMARY.md:46
 msgid "Manual Memory Management"
-msgstr ""
+msgstr "수동 메모리 관리"
 
-#: src/SUMMARY.md:49
+#: src/SUMMARY.md:47
 msgid "Scope-Based Memory Management"
-msgstr ""
+msgstr "범위기반 메모리 관리"
 
-#: src/SUMMARY.md:50
+#: src/SUMMARY.md:48
 msgid "Garbage Collection"
-msgstr ""
+msgstr "가비지 컬렉션"
 
-#: src/SUMMARY.md:51
+#: src/SUMMARY.md:49
 msgid "Rust Memory Management"
-msgstr ""
+msgstr "러스트의 메모리 관리"
 
-#: src/SUMMARY.md:52
+#: src/SUMMARY.md:50
 msgid "Comparison"
-msgstr ""
+msgstr "비교"
 
-#: src/SUMMARY.md:53
+#: src/SUMMARY.md:51
 msgid "Ownership"
-msgstr ""
+msgstr "소유권"
 
-#: src/SUMMARY.md:54
+#: src/SUMMARY.md:52
 msgid "Move Semantics"
-msgstr ""
+msgstr "의미 이동"
 
-#: src/SUMMARY.md:55
+#: src/SUMMARY.md:53
 msgid "Moved Strings in Rust"
-msgstr ""
+msgstr "러스트의 이동된 문자열"
 
-#: src/SUMMARY.md:56
+#: src/SUMMARY.md:54
 msgid "Double Frees in Modern C++"
-msgstr ""
+msgstr "모던C++에서 이중해제 문제"
 
-#: src/SUMMARY.md:57
+#: src/SUMMARY.md:55
 msgid "Moves in Function Calls"
-msgstr ""
+msgstr "함수호출에서 이동"
 
-#: src/SUMMARY.md:58
+#: src/SUMMARY.md:56
 msgid "Copying and Cloning"
-msgstr ""
+msgstr "복사와 복제"
 
-#: src/SUMMARY.md:59
+#: src/SUMMARY.md:57
 msgid "Borrowing"
-msgstr ""
+msgstr "빌림"
 
-#: src/SUMMARY.md:60
+#: src/SUMMARY.md:58
 msgid "Shared and Unique Borrows"
-msgstr ""
+msgstr "공유와 고유 빌림"
 
-#: src/SUMMARY.md:61
+#: src/SUMMARY.md:59
 msgid "Lifetimes"
-msgstr ""
+msgstr "수명"
 
-#: src/SUMMARY.md:62
+#: src/SUMMARY.md:60
 msgid "Lifetimes in Function Calls"
-msgstr ""
+msgstr "함수호출에서의 수명"
 
-#: src/SUMMARY.md:63
+#: src/SUMMARY.md:61
 msgid "Lifetimes in Data Structures"
-msgstr ""
+msgstr "자료 구조에서의 수명"
 
-#: src/SUMMARY.md:65
+#: src/SUMMARY.md:63
 msgid "Designing a Library"
-msgstr ""
+msgstr "도서관 설계"
 
-#: src/SUMMARY.md:66
+#: src/SUMMARY.md:64
 msgid "Iterators and Ownership"
-msgstr ""
+msgstr "반복자와 소유권"
 
-#: src/SUMMARY.md:69
+#: src/SUMMARY.md:67
 msgid "Day 2: Morning"
-msgstr ""
+msgstr "2일차 오전"
 
-#: src/SUMMARY.md:74
+#: src/SUMMARY.md:72
 msgid "Structs"
-msgstr ""
+msgstr "구조체"
 
-#: src/SUMMARY.md:75
+#: src/SUMMARY.md:73
 msgid "Tuple Structs"
-msgstr ""
+msgstr "튜플"
 
-#: src/SUMMARY.md:76
+#: src/SUMMARY.md:74
 msgid "Field Shorthand Syntax"
-msgstr ""
+msgstr "필드 할당 단축 문법"
 
-#: src/SUMMARY.md:77
+#: src/SUMMARY.md:75
 msgid "Enums"
-msgstr ""
+msgstr "열거형"
 
-#: src/SUMMARY.md:78
+#: src/SUMMARY.md:76
 msgid "Variant Payloads"
 msgstr ""
```

Comment:
데이터를 포함하는 열거형

Reply:
> > Hi @keispace . Thanks for translating https://google.github.io/comprehensive-rust/ into Korean first. Unfortunately, you and I have both worked on the same spot. What I've done on this PR is merging your translations and fixing templating issue(`msgfmt`) following @mgeisler 's comments above.
> 
> @joosangkim Hello, I am currently working on google/main modifications as of January 25th after the full translation draft has been completed. Not yet done but working if needed (fuzzy part needs to be worked) I'll share the [branch](https://github.com/keispace/comprehensive-rust-kr/tree/translate-ko). You can refer to the [ko.po](https://github.com/keispace/comprehensive-rust-kr/blob/translate-ko/po/ko.po) file.
> 
> @mgeisler My translation work will be completed around tomorrow, but I'm not sure if it's right for several Korean translation PRs to go up.
> 
> 안녕하세요. 관심있는 분이 또 계시니 반갑네요. 저는 현재 전체 번역 초안이 완료 되고 1월 25일 기준으로 google/main 수정사항을 작업 중입니다. 아직 완료는 되지 않았지만 필요하시면 작업중인(fuzzy 부분 및 4일차 키노트 미완료) [저장소](https://github.com/keispace/comprehensive-rust-kr/tree/translate-ko) 공유드리겠습니다.[ ko.po ](https://github.com/keispace/comprehensive-rust-kr/blob/translate-ko/po/ko.po)파일 참고하시면 될거 같습니다.
> 
> 제 번역 작업은 내일 쯤 완료 될텐데 한국어 번역 PR이 여러개 올라가는게 맞는지 모르겠네요

I am a Korean software engineer working for Android at Google and mgeisler@ suggested me to review this translation. Since we are chasing a moving target, I feel like we better submit this commit as is, and fix the remaining issues as follow-ups.